<launch>
	<arg name="pkg_name" value="micromvp_test"/>
	<arg name="node_name" default="subspeed_node"/>
	<arg name="veh" doc="Name of vehicle. ex: megaman"/>
	<arg name="tagID" doc="Subscribe specific tag ID. ex: tag39"/>
	<arg name="maxspeed" default="0.5" doc="Subscribe specific tag ID. ex: tag39"/>

	<remap from="subspeed" to="/gui_node/carspeeds"/>
	<rosparam param="tagID" subst_value="True">$(arg tagID)</rosparam>
	<rosparam param="maxspeed" subst_value="True">$(arg maxspeed)</rosparam>
	<node pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" clear_params="true" required="true"/>
	
</launch>